apps/req.c

```diff
@@ -626,7 +626,7 @@ int req_main(int argc, char **argv)
                        "         Your key size is %ld! Larger key size may behave not as expected.\n",
                        OPENSSL_RSA_MAX_MODULUS_BITS, newkey);
 
-#ifndef OPENSSL_NO_DSA
+#if !defined(OPENSSL_NO_DSA) || !defined(OPENSSL_NO_DEPRECATED_3_0)
```

Comment:
Why is this change needed?

Reply:
At this point, that's a very good question

Reply:
Undone